gnovm/pkg/gnolang/ownership.go

```diff
@@ -133,17 +133,29 @@ var (
 )
 
 type ObjectInfo struct {
-	ID        ObjectID  // set if real.
-	Hash      ValueHash `json:",omitempty"` // zero if dirty.
-	OwnerID   ObjectID  `json:",omitempty"` // parent in the ownership tree.
-	ModTime   uint64    // time last updated.
-	RefCount  int       // for persistence. deleted/gc'd if 0.
-	IsEscaped bool      `json:",omitempty"` // hash in iavl.
+	ID       ObjectID  // set if real.
+	Hash     ValueHash `json:",omitempty"` // zero if dirty.
+	OwnerID  ObjectID  `json:",omitempty"` // parent in the ownership tree.
+	ModTime  uint64    // time last updated.
+	RefCount int       // for persistence. deleted/gc'd if 0.
+
+	// Object has multiple references (refcount > 1) and is persisted separately
+	IsEscaped bool `json:",omitempty"` // hash in iavl.
+
 	// MemRefCount int // consider for optimizations.
-	isDirty      bool
-	isDeleted    bool
-	isNewReal    bool
+	// Object has been modified and needs to be saved
+	isDirty bool
+
+	// Object has been permanently deleted
+	isDeleted bool
+
+	// Object is newly created in current transaction and will be persisted
+	isNewReal bool
+
+	// Object newly created multiple references in current transaction
 	isNewEscaped bool
+
+	// Object is marked for deletion in current transaction
 	isNewDeleted bool
 
 	// XXX huh?
```

Comment:
nit: maybe take the opportunity to fix this? It's quite confusing why it's here.

Reply:
Jae might remember what it is for when he looks at it.
If we delete it, its purpose might be forgotten completely.